packages/components/organisms/f-header/src/components/Logo.vue

```diff
@@ -41,6 +45,10 @@ export default {
             type: String,
             required: true
         },
+        linkDisabled: {
```

Comment:
Are you planning to use the same prop for other links too? If not I would just called it something like `isLogoDisabled`